plugins/drop_folder/lib/kDropFolderEventsConsumer.php

```diff
@@ -266,6 +266,10 @@ private function onEntryStatusChanged( $entry )
 	{
 		// Handle only files that are still in the PROCESSING state, which were left
 		// in this state due to AUTO_DELETE_WHEN_ENTRY_IS_READY delete policy.
+        if ($entry->getIsTemporary())
+        {
+            $entry = $entry->getReplacedEntryId();
+        }
```

Comment:
@OrelAri I think this is fragile + you place entryId value into what was an $entry object
Better to

Please consider using the fix I suggested when we investigated it:
if ($entry->getIsTemporary() && $entry->getReplacedEntryId())
        {
            $entryId = $entry->getReplacedEntryId();
        }
        else
        {
            $entryId = $entry->getId();
        }